internal/repository/prisma/tenant_alerting.go

```diff
@@ -142,22 +147,48 @@ func (r *tenantAlertingEngineRepository) GetTenantAlertingSettings(ctx context.C
 		return nil, err
 	}
 
+	groupsForSend := make([]*repository.TenantAlertEmailGroupForSend, 0)
+
 	emailGroups, err := r.queries.GetEmailGroups(ctx, tx, pgTenantId)
 
 	if err != nil {
 		return nil, err
 	}
 
+	for _, group := range emailGroups {
+		groupsForSend = append(groupsForSend, &repository.TenantAlertEmailGroupForSend{
+			TenantId: group.TenantId,
+			Emails:   group.Emails,
+		})
+	}
+
 	tenant, err := r.queries.GetTenantByID(ctx, tx, pgTenantId)
 
 	if err != nil {
 		return nil, err
 	}
 
+	if tenant.AlertMemberEmails {
+		emails, err := r.queries.GetMemberEmailGroup(ctx, tx, pgTenantId)
```

Comment:
I actually looks like this tx is never committed, is that right? Should fix ASAP.

Reply:
good catch